### ISSUE TITLE
Broken news links contrast too low

### ISSUE THREAD

**Issue by patricoferris** [ocaml/v2.ocaml.org#1244]
On the [Planet news feed page](https://ocaml.org/community/planet/) there is a list of syndications. Some of these are now broken and are presented with a strikethrough and very low contrast. The contrast is in fact too low to meet WCAG standards for accessibility in particular [item 1.4.3 of the WCAG 2.1 standard](https://www.w3.org/TR/WCAG21/#contrast-minimum). The list item in the red box is an example. 

<img width="311" alt="broken-link" src="https://user-images.githubusercontent.com/20166594/109646595-4e86f680-7b50-11eb-96eb-ba2cd2eaf46e.png">

I believe the strikethrough is sufficient enough (testing using vision deficiencies emulation using the chrome browser suggests to me this is enough). 

@nayaldivya if you would like, to this might a great first issue for you to contribute too? No pressure if you don't want to this is just in relation to [your comment](https://github.com/ocaml/ocaml.org/pull/1241#issuecomment-788187415) on #1241. If you could add a message if you would like to look into this or not in the not too distant future so other people know if this issue is open or not that would be fantastic. Thank you for wanting to contribute to the site :))

(If you do take this up feel free to ask any questions here on this issue or on a PR that you open in relation to it).

**Comment by nayaldivya** [comment]
Thanks @patricoferris. I would love to work on this.

**Comment by IIITM-Jay** [comment]
@nayaldivya Are you still working on this...

@patricoferris This is Jay  here, may I request any other issue  on which I can work on... (or I can submit a PR on this issue)

**Comment by nayaldivya** [comment]
> @nayaldivya Are you still working on this...
> Yes.

**Comment by patricoferris** [comment]
@IIITM-Jay please don't submit a PR for this issue. I have just created #1245 which, going forward, will be the place to register interest. I know you are interested and in due course I will find/create an issue and ping you about it.

@nayaldivya please don't feel any pressure to rush your contribution, take as much time as you need and don't hesitate to ask any questions on here.

**Comment by nayaldivya** [comment]
![ocaml_error](https://user-images.githubusercontent.com/73695161/109870300-f0612d00-7c8f-11eb-8e75-da4b3f8f51ea.png)
Hey @patricoferris, while setting up this project on my system, I came across this error. I'm using Windows 10 as my OS.

**Comment by nayaldivya** [comment]
Hey @patricoferris, while setting up the project on my system, I ran into this error. I'm using Windows 10 as my Operating System. Kindly have a look at this.
![ocaml_error](https://user-images.githubusercontent.com/73695161/109872440-6feffb80-7c92-11eb-8a7a-aad4009e9acd.png)

This is after I ran `opam install ocamlorg --deps-only` command.

**Comment by nayaldivya** [comment]
> Hey @patricoferris, while setting up the project on my system, I ran into this error. I'm using Windows 10 as my Operating System. Kindly have a look at this.
> ![ocaml_error](https://user-images.githubusercontent.com/73695161/109872440-6feffb80-7c92-11eb-8a7a-aad4009e9acd.png)

This is after I ran `opam install ocamlorg --deps-only` command.

**Comment by nayaldivya** [comment]
To resolve this issue #1244, I believe
In ocamlorg.css 
`.subscribers .broken {
  color: #dcc8c4;
  text-decoration: line-through;
}`
should be updated to 
`.subscribers .broken {
  text-decoration: line-through;
}`

This will change 

![syndications_before](https://user-images.githubusercontent.com/73695161/109878832-cd884600-7c9a-11eb-9c00-62c21947d78e.png)
to
![syndications_after](https://user-images.githubusercontent.com/73695161/109878860-d8db7180-7c9a-11eb-8d6d-9476cde3e628.png)

Kindly confirm this.

**Comment by patricoferris** [comment]
Great @nayaldivya  -- that looks about right to me. 

However, I would really love for you to get this up and running locally especially because you are developing on windows and any problems and solutions you find, we can add to the [contributing document](https://github.com/ocaml/ocaml.org/pull/1241) to help other windows devs.

I most confess windows support for OCaml isn't as good as other OSes but it looks like you have the harder bits already sorted. To start can I ask which instructions you followed to get your windows environment set up ?

**Comment by nayaldivya** [comment]
I followed this link: http://fdopen.github.io/opam-repository-mingw/installation/ After running the graphical installation file, it automatically installed OCaml, OPam, and Cygwin on the system.

**Comment by patricoferris** [comment]
This was fixed in #1252, thanks @nayaldivya :))
